### ISSUE TITLE
cargo web start doesn't rebuild JS when prepend-js changes.

### ISSUE THREAD

**Issue by kevincox** [koute/cargo-web#134]
I added `prepend-js = "static/myfile.js"` and modifications to this file do not cause a rebuild. In order to get a fresh build I need to kill and restart the server.

A work-around is to remove `prepend-js` and include the file manually but then it isn't available for tests.

**Comment by koute** [comment]
Yep. This is more-or-less a known problem; see `paths_to_watch` in `build.rs`. Fixing this requires two things: 1) generating the relevant paths in `paths_to_watch`, 2) making `cargo-web` always regenerate the `.js` file (see `process_wasm_file` in `wasm.rs`).
